drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -160,16 +173,16 @@ def items_without_answers():
             "url_name": getattr(self, 'url_name', ''),
             "display_zone_labels": self.data.get('displayLabels', False),
             "display_zone_borders": self.data.get('displayBorders', False),
-            "items": items_without_answers(),
-            "title": self.display_name,
+            "items": items_without_answers(self),
+            "title": self._(self.display_name),
```

Comment:
This is an instance of a translatable (Text?) field, that I would want to get translated by the XBlock runtime (with translations being provided by the author)